docs/tutorial/vald-agent-standalone-on-k8s.md

```diff
@@ -333,49 +335,48 @@ This chapter uses [NGT](https://github.com/yahoojapan/ngt) as Vald Agent to perf
 
     1.  Remove
 
-            - Remove indexed 400 training datasets from the Vald agent.
-                <details><summary>example code</summary><br>
-
-                ```go
-                for i := range ids [:insertCount] {
-                    _, err := client.Remove(ctx, &payload.Remove_Request{
-                        Id: &payload.Object_ID{
-                            Id: ids[i],
-                        },
-                    })
-                    if err != nil {
-                        glg.Fatal(err)
-                    }
-                    if i%10 == 0 {
-                        glg.Infof("Removed %d", i)
-                    }
-                }
-                ```
-
-                </details>
-
-
-            - Remove from the index manually instead of waiting for auto indexing.
-            The removed vectors still exist in the NGT graph index before the SaveIndex (or CreateAndSaveIndex) API is called.
-            If you run the below code, the indexes will be removed completely from the Vald Agent NGT graph and the Backup file.
-                <details><summary>example code</summary><br>
-
-                ```go
-                _, err = client.SaveIndex(ctx, &payload.Empty{})
-                if err != nil {
-                    glg.Fatal(err)
-                }
-                ```
-
-                </details>
+        - Remove indexed 400 training datasets from the Vald agent.
```

Comment:
📝 I fixed the indentation because it is broken.
👉 <img width="1157" alt="スクリーンショット 298" src="https://github.com/vdaas/vald/assets/25459661/0aa98af2-aa6a-42c2-9fe0-49df243375d9">